Cargo.toml

```diff
@@ -112,10 +127,9 @@ rtrb = { version = "0.2.2", optional = true }
 # Used for the `vst3` feature
 vst3-sys = { git = "https://github.com/robbert-vdh/vst3-sys.git", branch = "fix/drop-box-from-raw", optional = true }
 
-# Used for the `zstd` feature
 zstd = { version = "0.12.3", optional = true }
 
-[dev-dependencies]
+[dev-dependencies] # Used for the `zstd` feature
```

Comment:
You may want to disable whatever auto formatter you use for TOML because it's doing some funky things. :wink: